debug/index.js

```diff
@@ -0,0 +1,3 @@
+if (process.env!=='production') {
```

Comment:
I think it should be `process.env.NODE_ENV` 👍

Reply:
Oh yeah, still have to test it aswell so that would’ve showed ^^